Code/Editor/Util/ImageUtil.cpp

```diff
@@ -143,7 +143,8 @@ bool CImageUtil::LoadPGM(const QString& fileName, CImageEx& image)
     fseek(file, 0, SEEK_SET);
 
     char* str = new char[fileSize];
-    fread(str, fileSize, 1, file);
+
+    [[maybe_unused]] auto bytesRead = fread(str, fileSize, 1, file);
```

Comment:
What is the warning flag that makes these C functions `[[nodiscard]]`?

Reply:
`-Wunused-result`. This shows up on GCC if I remove this warning disable, but for some reason one of the security flags convinced clang of this issue as well.